src/Configuration/Configuration.php

```diff
@@ -51,13 +51,18 @@ public function set(string $name, $value): void
 
     public function has(string $name): bool
     {
-        $ok = array_key_exists($name, $this->values);
-        if ($ok) {
+        if (array_key_exists($name, $this->values)) {
             return true;
         }
+
         if ($this->parent) {
             return $this->parent->has($name);
         }
+
+        if (array_key_exists($name, \getenv())) {
```

Comment:
I don’t like what it’s for all the configurations. Let’s make it optional only in yaml via explicit exporfs.

Reply:
What do you mean by `via explicit exporfs`?

Actually, it would also be usefull for PHP configs too as in that way you don't need to edit your script to modulate its behavior and thus you can share your config across your projects. The way it is implemented is transparent for actual users who would upgrade to this version (because it is a fallback), but we can add a parameter to explicitly enable this feature to be sure to avoid any side effect.

Reply:
@antonmedv @Schrank any update on this please ?